src/Api/Groups.php

```diff
@@ -144,6 +144,21 @@ public function members($group_id, array $parameters = [])
         return $this->get('groups/'.self::encodePath($group_id).'/members', $resolver->resolve($parameters));
     }
 
+    /**
+     * @return mixed
```

Comment:
This PR cannot be merged without proper phpdoc. Can you please add the necessary param doc.